package.json

```diff
@@ -8,7 +7,8 @@
         "watch": "lerna run watch",
         "start": "lerna run start",
         "test": "lerna run test --",
-        "publish:latest": "lerna publish --registry=https://registry.npmjs.org/ --exact --no-git-tag-version --no-push",
+        "publish:latest:ci": "lerna run publish:latest:ci",
```

Comment:
I could not get lerna to publish a "latest" without prompting for the user to select a version. So instead I call a script from the deeper package.json, and there use "yarm publish --non-interactive [...]" to accomplish theat goal.

Reply:
no longer relevant